src/Tools/AI Test Toolkit/src/AITTestContext.Codeunit.al

```diff
@@ -0,0 +1,312 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace System.TestTools.AITestToolkit;
+using System.TestTools.TestRunner;
+
+/// <summary>
+/// Exposes functions that can be used by the AIT tests.
+/// </summary>
+codeunit 149043 "AIT Test Context"
```

Comment:
This contains public, internal and local methods, are we not going to follow a facade pattern?

Reply:
We will see to refactor this one. The issue is that the codeunit is static. I'm hesitant to introduce 2 static codeunits nowt.